src/common-principles.md

```diff
@@ -409,20 +409,22 @@ datasets and non-compliant derivatives.
 
 ### Imaging files
```

Comment:
Wondering if this heading should not be renamed now that this section has become more generic.

Reply:
I used Imaging files as generic file, including for example EEG. I can't find a good name, data file will also incompass the tabular files. But I'm open for suggestions.

Reply:
yeah I was more making a note also for other reviewers